docs/demand/faf.rst

```diff
@@ -21,12 +21,20 @@ datasets based on the relative tonnage of known modes (truck, rail, water) for e
 This proportional allocation ensures that demand for unknown modes is captured according to the
 transportation patterns of known modes.
 
+County-to-County Processing
+---------------------------
+The demand data is processed to generate county-to-county origin-destination pairs for each mode of transportation
+(truck, rail, and water). This involves allocating FAF zone-level demand to counties using a predefined allocation map,
```

Comment:
I think it's worth putting a bit more detail in here...and I even have some slides that explain this.
* Look up population data in each county
* Compute the area of each county within each FAF region
* Compute the total population within each FAF region (see uniform assumption, above)
* Proportionally allocate FAF demand into each county, and localize it in each county's centroid.

Thus, each county has **one** geographic point (the county centroid) where demand originates/terminates for each mode.